code/datums/mind.dm

```diff
@@ -265,6 +265,15 @@
 		P = locate() in PDA
 	if (!P) // If we couldn't find a pen in the PDA, or we didn't even have a PDA, do it the old way
 		P = locate() in all_contents
+		if(!P) // I do not have a pen.
+			if(traitor_mob.back) //ok buddy you better have a backpack!
+				inowhaveapen = new var/obj/item/pen(traitor_mob.back)

```

Comment:
You removed the wrong `var/`! 😛 
This line just needs to be `inowhaveapen  new /obj/item/pen(traitor_mob.back)`, and you also need to name the var *outside* of the new call. The best way to do this is to put it between lines 267 and 267 as `var/obj/item/pen/inowhaveapen`, since you already define it in the if/else calls.

Reply:
well fuck me